### DISCUSSION TITLE
Better tmux status bar rendering in kitty

### DISCUSSION THREAD

**Discussion by pauljacobson** [kovidgoyal/kitty#4654] [Q&A] [answered]
Hi there 👋 I'm experimenting with kitty, and I noticed that when I join a tmux session in kitty, my status bar isn't rendering as I'd expect:

![kitty_tmux](https://user-images.githubusercontent.com/2912202/152950743-78607bab-c77f-4c61-af61-ab610d9a6751.png)

By contrast, this is what I have in iTerm2:

![iterm_tmux](https://user-images.githubusercontent.com/2912202/152950802-0cf2a498-6608-446e-a886-096d33819c68.png)

I'm pretty sure I have the same font setup in kitty. My question is where I should be looking in my kitty config (or elsewhere) to troubleshoot this issue? I'm not entirely where to look to resolve this issue.

**Comment by page-down** [comment, marked as answer]
Don't use patch fonts. Please see the following link for the standalone nerd fonts configuration.

https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

Use the following command to check the font.

```shell
kitty --debug-font-fallback
```

https://sw.kovidgoyal.net/kitty/invocation/#cmdoption-kitty-debug-font-fallback

**Reply by pauljacobson** [reply]
Thank you. I'll take a closer look at that.
